<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>550</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LiteChat-用户注册</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>-20</y>
     <width>511</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>造字工房致黑（非商用）常规体</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>用户注册</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="account">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>造字工房致黑（非商用）常规体</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="set_account">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solid #404040; /* 边框宽度为1px，颜色为#A0A0A0 */
	border-radius: 8px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: #FFFFFF; /* 背景颜色 */
	color: #0A0A0A; /* 文本颜色 */
	selection-background-color: #298DFF; /* 选中文本的背景颜色 */
	selection-color: #F2F2F2; /* 选中文本的颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	font-size: 10pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 2px solid #00EE76;
	border-radius: 8px;
	background-color: #FFFFFF;
        color: #0A0A0A;
        selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="set_password">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solid #404040; /* 边框宽度为1px，颜色为#A0A0A0 */
	border-radius: 8px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: #FFFFFF; /* 背景颜色 */
	color: #0A0A0A; /* 文本颜色 */
	selection-background-color: #298DFF; /* 选中文本的背景颜色 */
	selection-color: #F2F2F2; /* 选中文本的颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	font-size: 10pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 2px solid #00EE76;
	border-radius: 8px;
	background-color: #FFFFFF;
        color: #0A0A0A;
        selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="password">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>造字工房致黑（非商用）常规体</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="nickname">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>造字工房致黑（非商用）常规体</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>昵称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="set_nickname">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solid #404040; /* 边框宽度为1px，颜色为#A0A0A0 */
	border-radius: 8px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: #FFFFFF; /* 背景颜色 */
	color: #0A0A0A; /* 文本颜色 */
	selection-background-color: #298DFF; /* 选中文本的背景颜色 */
	selection-color: #F2F2F2; /* 选中文本的颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	font-size: 10pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 2px solid #00EE76;
	border-radius: 8px;
	background-color: #FFFFFF;
        color: #0A0A0A;
        selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="signature">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>造字工房致黑（非商用）常规体</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>个性签名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="set_signature">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solid #404040; /* 边框宽度为1px，颜色为#A0A0A0 */
	border-radius: 8px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: #FFFFFF; /* 背景颜色 */
	color: #0A0A0A; /* 文本颜色 */
	selection-background-color: #298DFF; /* 选中文本的背景颜色 */
	selection-color: #F2F2F2; /* 选中文本的颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	font-size: 10pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 2px solid #00EE76;
	border-radius: 8px;
	background-color: #FFFFFF;
        color: #0A0A0A;
        selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_register">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>440</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    color:#ffffff; /*文字颜色*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #f6d365, stop: 1 #fda085);/*背景色*/
    border-style:outset; /*边框风格*/
    border-width:2px;/*边框宽度*/
    border-color:#FFCC66; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
    font:Bold 25px; /*字体*/
    font-family: &quot;Microsoft YaHei&quot;;
    min-width:100px;/*控件最小宽度*/
    min-height:20px;/*控件最小高度*/
    padding:4px;/*内边距*/
}
 
QPushButton:hover
{
    color:#ffffff; /*文字颜色*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #fda085, stop: 1 #fee140);/*背景色*/
    border-style:outset; /*边框风格*/
    border-width:2px;/*边框宽度*/
    border-color:#FFCC66; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
    font:bold 25px; /*字体*/
    font-family: &quot;Microsoft YaHei&quot;;
    min-width:100px;/*控件最小宽度*/
    min-height:20px;/*控件最小高度*/
    padding:4px;/*内边距*/
}
QPushButton:pressed
{
    color:#ffffff; /*文字颜色*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #fda085, stop: 1 #43e97b);/*背景色*/
    border-style:outset; /*边框风格*/
    border-width:2px;/*边框宽度*/
    border-color:#FFCC66; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
    font:bold 25px; /*字体*/
    font-family: &quot;Microsoft YaHei&quot;;
    min-width:100px;/*控件最小宽度*/
    min-height:20px;/*控件最小高度*/
    padding:4px;/*内边距*/
}



</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_login">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    color:#ffffff; /*文字颜色*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #f6d365, stop: 1 #fda085);/*背景色*/
    border-style:outset; /*边框风格*/
    border-width:2px;/*边框宽度*/
    border-color:#FFCC66; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
    font:Bold 25px; /*字体*/
    font-family: &quot;Microsoft YaHei&quot;;
    min-width:100px;/*控件最小宽度*/
    min-height:20px;/*控件最小高度*/
    padding:4px;/*内边距*/
}
 
QPushButton:hover
{
    color:#ffffff; /*文字颜色*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #fda085, stop: 1 #fee140);/*背景色*/
    border-style:outset; /*边框风格*/
    border-width:2px;/*边框宽度*/
    border-color:#FFCC66; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
    font:bold 25px; /*字体*/
    font-family: &quot;Microsoft YaHei&quot;;
    min-width:100px;/*控件最小宽度*/
    min-height:20px;/*控件最小高度*/
    padding:4px;/*内边距*/
}
QPushButton:pressed
{
    color:#ffffff; /*文字颜色*/
    background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop:0 #fda085, stop: 1 #43e97b);/*背景色*/
    border-style:outset; /*边框风格*/
    border-width:2px;/*边框宽度*/
    border-color:#FFCC66; /*边框颜色*/
    border-radius:10px; /*边框倒角*/
    font:bold 25px; /*字体*/
    font-family: &quot;Microsoft YaHei&quot;;
    min-width:100px;/*控件最小宽度*/
    min-height:20px;/*控件最小高度*/
    padding:4px;/*内边距*/
}



</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="login">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>造字工房致黑（非商用）常规体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;&lt; 已有账户？前往登录 </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="retype_password">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solid #404040; /* 边框宽度为1px，颜色为#A0A0A0 */
	border-radius: 8px; /* 边框圆角 */
	padding-left: 5px; /* 文本距离左边界有5px */
	background-color: #FFFFFF; /* 背景颜色 */
	color: #0A0A0A; /* 文本颜色 */
	selection-background-color: #298DFF; /* 选中文本的背景颜色 */
	selection-color: #F2F2F2; /* 选中文本的颜色 */
	font-family: &quot;Microsoft YaHei&quot;; /* 文本字体族 */
	font-size: 10pt; /* 文本字体大小 */
}

QLineEdit:hover { /* 鼠标悬浮在QLineEdit时的状态 */
	border: 2px solid #00EE76;
	border-radius: 8px;
	background-color: #FFFFFF;
        color: #0A0A0A;
        selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="retype">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>造字工房致黑（非商用）常规体</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>再次输入密码</string>
   </property>
  </widget>
  <widget class="QToolButton" name="close">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>×</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="minimize">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>550</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>15</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pic/theychat-mini.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hint_1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*请输入4~9位数字</string>
   </property>
  </widget>
  <widget class="QLabel" name="hint_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>220</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*请输入4~16位英文字母与数字的组合</string>
   </property>
  </widget>
  <widget class="QLabel" name="hint_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*请再次确认您的密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="hint_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>*默认昵称与账号相同，可以输入汉字，字母，符号或它们的组合</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>title</zorder>
  <zorder>account</zorder>
  <zorder>set_account</zorder>
  <zorder>set_password</zorder>
  <zorder>password</zorder>
  <zorder>nickname</zorder>
  <zorder>set_nickname</zorder>
  <zorder>signature</zorder>
  <zorder>set_signature</zorder>
  <zorder>go_register</zorder>
  <zorder>go_login</zorder>
  <zorder>login</zorder>
  <zorder>retype_password</zorder>
  <zorder>retype</zorder>
  <zorder>close</zorder>
  <zorder>minimize</zorder>
  <zorder>logo</zorder>
  <zorder>hint_1</zorder>
  <zorder>hint_2</zorder>
  <zorder>hint_3</zorder>
  <zorder>hint_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
